docs/plugindev/README.md

```diff
@@ -232,6 +232,34 @@ make inventory-init
 make inventory-plugin-add
 ```
 
+### Updating the plugin group
+
+Create `plugin-scope-association.yaml` under `$(ROOT_DIR)/cmd/plugins/plugin-scope-association.yaml` of plugin repository
```

Comment:
As a user, I don't know what `$ROOT_DIR` is.  Also the `$()` is a makefile-specific variable access.  It does not work in the shell as it instead calls a subshell.

```suggestion
Create a `plugin-scope-association.yaml` file under `cmd/plugin/plugin-scope-association.yaml` of your plugin repository.
```

Notice that while testing I saw that the directory is `cmd/plugin` instead of `cmd/plugins` when developing a new plugin project.